plugins/rapid7_insightidr/unit_test/test_search_alerts.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from komand_rapid7_insightidr.actions.search_alerts import SearchAlerts
+from komand_rapid7_insightidr.actions.search_alerts.schema import (
+    Input,
+    SearchAlertsInput,
+    SearchAlertsOutput,
+)
+from util import Util
+from unittest.mock import patch
+from parameterized import parameterized
+from insightconnect_plugin_runtime.exceptions import PluginException
+from jsonschema import validate
+
+
+@patch("requests.Session.request", side_effect=Util.mocked_requests)
+class TestSearchAlerts(TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.action = Util.default_connector(SearchAlerts())
+
+    @parameterized.expand(Util.load_parameters("search_alerts").get("parameters"))
+    def test_search_alerts(
+        self,
+        mock_request,
+        start_time,
+        end_time,
+        leql,
+        terms,
+        sorts,
+        field_ids,
+        aggregates,
+        size,
+        index,
+        rrns_only,
+        expected,
+    ):
```

Comment:
I suggest adding typehints in each test method

Reply:
type hints have been added to the unit tests